enigmatry-entry-blueprint-app/package.json

```diff
@@ -60,6 +60,6 @@
     "npm-audit-reporter-teamcity": "0.4.0",
     "nswag": "^14.0.0",
     "ts-node": "~10.4.0",
-    "typescript": "4.9.5"
+    "typescript": "5.1.6"
```

Comment:
Angular 16 support typescript >=4.9.3 <5.2.0. We can use latest supported version.